File: universityClient/src/app/Admin/_adminModels/sectionsParams.ts
export class SectionsParams {
  pageNumber = 1;
  pageSize = 12;
  CourseId: number;
  MajorId: number;
  FacultyId: number;
  TeacherId: number;

  constructor(
    CourseId?: number,
    MajorId?: number,
    FacultyId?: number,
    TeacherId?: number
  ) {
    this.FacultyId = FacultyId;
    this.MajorId = MajorId;
    this.CourseId = CourseId;
    this.TeacherId = TeacherId;
  }
}

File: university APIs/Helpers/AdminHelpers/SectionsParams.cs
namespace university_APIs.Helpers.AdminHelpers
{
    public class SectionsParams:Params
    {
        public int? CourseId { get; set; }
        public int? MajorId{ get; set; }
        public int? FacultyId { get; set; }
        public int? TeacherId{ get; set; }
    }
}
